ambari-common/src/main/python/resource_management/libraries/execution_command/execution_command.py

```diff
@@ -54,6 +54,15 @@ def __get_value(self, key, default_value=None):
     except:
       return default_value
 
+  def query_config_attri_directly(self, query_string, default_value=None):
+    """
+    Query config attribute from execution_command directly, i.e "clusterHostInfo/zookeeper_server_hosts"
```

Comment:
A better name would be `get_value`, since this method returns exactly the same as `__get_value`, the only difference is that it is public.